masterlist.yaml

```
@@ -9660,19 +9660,93 @@ plugins:
         itm: 6
 
   - name: 'BetterInns .*\.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/29333/' ]
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/29333/'
+        name: 'BetterInns_and_NewHomeOwners'
   - name: 'BetterInns + NewHomeOwners.esp'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Oblivion Character Overhaul'
+          - '[OCOv2 - Filter Patch for Mods](https://www.nexusmods.com/oblivion/mods/51379/)'
+        condition: 'active("Bashed Patch.*\.esp") and active("Oblivion_Character_Overhaul.esp") and not file("OCOv2 - Filter Patch for Mods.esp")'
+      - <<: *patchProvided
+        subs: [ 'Black Horse Courier Expanded' ]
+        condition: 'active("BHC_Expanded.esp") and not active("BetterInns - BHCExpanded PATCH.esp")'
+      - <<: *patchProvided
+        subs: [ 'Clocks of Cyrodiil' ]
+        condition: 'active("ClocksOfCyrodiil.esp") and not active("BetterInns - ClocksOfCyrodiil PATCH.esp")'
+      - <<: *patchProvided
+        subs: [ 'Cobl' ]
+        condition: 'active("Cobl Glue.esp") and not active("BetterInns - COBL PATCH.esp")'
+      - <<: *patchProvided
+        subs: [ 'Harvest Containers' ]
+        condition: 'active("Harvest[Containers].esp") and not file("BetterInns - Harvest Containers PATCH.esp")'
+      - <<: *patchProvided
+        subs: [ 'Salmo the Baker' ]
+        condition: 'active("Salmo the Baker, Cobl.esp") and not file("BetterInns - Salmo the Baker, Cobl PATCH.esp")'
+      - <<: *patchProvided
+        subs: [ 'Salmo the Baker' ]
```

It'd help to differentiate between the two patch messages to avoid confusion about which patch is being recommended to the user.

```suggestion
        subs: [ 'Salmo the Baker Cobl' ]
        condition: 'active("Salmo the Baker, Cobl.esp") and not file("BetterInns - Salmo the Baker, Cobl PATCH.esp")'
      - <<: *patchProvided
        subs: [ 'Salmo the Baker v2' ]
```